tasks/README.md

```diff
@@ -67,6 +67,10 @@ Name | Summary | Category
 `task060_ropes_question_generation`	| Constructing questions regarding relations in the given paragraph.	| Question Generation
 `task061_ropes_answer_generation`	| Answering questions regarding relations in the given paragraph.	| Answer Generation
 `task062_bigbench_repeat_copy_logic`	|  Generating text that follows simple logical operations such as "repeat", "before", "after" etc.	| Logic
+`task063_timetravel_text_modification`	| Modifying a sentence in the given story to make the story inconsistent. | Text Modification
+`task065_timetravel_classification`	| Choosing the option that makes a given short story consistent. | Classification
+`task066_timetravel_classification`	| Identifying if the given sentence is consistent with the given story. | Classification
```

Comment:
Make the titles more informative? Something like this: 
- `task065_timetravel_consistent_sentence_classification `
- `task066_timetravel_binary_consistency_classification`